optaplanner-core/src/test/java/org/optaplanner/core/impl/partitionedsearch/queue/PartitionQueueTest.java

```diff
@@ -31,6 +33,16 @@
 
     private ExecutorService executorService = Executors.newFixedThreadPool(2);
 
+    @After
+    public void tearDown() {
+        executorService.shutdown();
+        try {
+            executorService.awaitTermination(1, TimeUnit.SECONDS);
+        } catch (InterruptedException ex) {
+            throw new RuntimeException(ex);
```

Comment:
InterruptedException should never be handled this way because it eats the interrupted flag. Look for other catch InterruptedException's in the code on how to handle it properly.